HopsanCLI/CMakeFiles/CMakeDirectoryInformation.cmake

```diff
@@ -0,0 +1,16 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
You accidentally added and committed the CMake build cache, it should be removed

Reply:
I pushed a fixup commit to remove these files, but you need to squash them